custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -125,9 +124,10 @@ def get_person_case_structure(self):
             },
         }
 
-        if self._location:
-            if self._location.location_type.code == 'phi':
-                kwargs['attrs']['owner_id'] = self._location.location_id
+        if self.phi:
+            if self.phi.location_type.code == 'phi':
+                kwargs['attrs']['owner_id'] = self.phi.location_id
+                kwargs['attrs']['update']['phi'] = self.phi.name
```

Comment:
It looks like ```phi``` is the name, not the location_id. ([example](https://india.commcarehq.org/a/enikshay-test/reports/case_data/577ac64b-b909-4cad-9cac-c31d1888b585/))  Am I understanding that correctly?